pegasus/sites.v3/code.org/public/teacher-dashboard/index.haml

```diff
@@ -68,10 +68,6 @@ title: <%= I18n.t('dashboard_landing_title').inspect %>
 
 #gdpr-dialog
 
-- provider = dashboard_user[:provider]
-- provider = 'google_classroom' if provider == 'google_oauth2'
-- data[:provider] = provider
-
 /We Intentionally take the non-minified JS file here, as it contains a bunch of
```

Comment:
This is replaced with the relevant section's login type in [teacher-dashboard/index.js](https://github.com/code-dot-org/code-dot-org/blob/4c5e8727c593fbbeb1e06f52bfeef4a1135ccec9/apps/src/sites/code.org/pages/public/teacher-dashboard/index.js#L421) since the user can have many providers but the re-sync button (and other roster import logic) is only relevant to a section.

Reply:
Oh, I see.  Great!